doc/dev_start_guide.txt

```diff
@@ -267,41 +252,37 @@ You can use ``easy_install`` instead of ``pip``, and ``pep8`` instead of
 ``flake8`` if you prefer. The important thing is that the ``flake8`` or
 ``pep8`` executable ends up in your ``$PATH``.
 
-To install Syntastic, according to its documentation, the easiest way is
-to install `pathogen.vim`_ first.
-
-Here's a relevant extract of pathogen.vim's installation instructions:
-
-    Install to ``~/.vim/autoload/pathogen.vim``. Or copy and paste::
-
-        mkdir -p ~/.vim/autoload ~/.vim/bundle; \
-        curl -so ~/.vim/autoload/pathogen.vim \
-            https://raw.github.com/tpope/vim-pathogen/HEAD/autoload/pathogen.vim
+Install vundle with: ``git clone https://github.com/VundleVim/Vundle.vim.git
```

Comment:
To highlight the command you should put it in a indented line with an empty line before and after, like was done for flake8 [here](https://github.com/Theano/Theano/pull/4189/files#diff-4e41d42674d7822d1b2f6461f1b067b2R249).